2025-04-09 16:10:32,365 INFO    StreamThr :3613517 [internal.py:wandb_internal():86] W&B internal server running at pid: 3613517, started at: 2025-04-09 16:10:32.365052
2025-04-09 16:10:32,366 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status
2025-04-09 16:10:32,370 INFO    WriterThread:3613517 [datastore.py:open_for_write():85] open: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/4/wandb/run-20250409_161032-65xw8huw/run-65xw8huw.wandb
2025-04-09 16:10:32,370 DEBUG   SenderThread:3613517 [sender.py:send():380] send: header
2025-04-09 16:10:32,373 DEBUG   SenderThread:3613517 [sender.py:send():380] send: run
2025-04-09 16:10:32,495 ERROR   SenderThread:3613517 [internal_api.py:execute():369] 403 response executing GraphQL.
2025-04-09 16:10:32,496 ERROR   SenderThread:3613517 [internal_api.py:execute():370] {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-04-09 16:10:32,496 ERROR   SenderThread:3613517 [sender.py:send_run():977] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 975, in send_run
    self._init_run(run, config_value_dict)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2007, in upsert_run
    response = self.gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 964, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-04-09 16:10:35,426 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:35,967 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:36,407 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:36,836 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:37,373 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:10:37,497 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:40,427 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:40,967 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:41,407 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:41,837 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:42,374 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:10:42,497 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:45,427 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:45,968 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:46,408 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:46,837 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:47,375 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:10:47,498 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:50,428 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:50,969 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:51,409 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:51,838 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:52,376 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:10:52,499 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:55,428 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:55,969 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:56,409 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:56,838 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:10:57,377 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:10:57,499 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:00,429 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:00,970 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:01,410 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:01,839 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:02,378 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:02,500 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:05,430 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:05,970 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:06,411 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:06,840 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:07,379 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:07,500 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:10,430 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:10,971 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:11,412 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:11,840 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:12,380 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:12,501 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:15,431 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:15,971 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:16,412 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:16,841 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:17,380 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:17,501 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:20,431 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:20,972 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:21,413 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:21,841 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:22,382 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:22,502 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:25,432 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:25,973 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:26,413 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:26,842 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:27,382 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:27,502 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:30,432 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:30,973 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:31,414 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:31,843 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:32,383 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:32,503 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:35,433 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:35,974 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:36,415 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:36,843 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:37,384 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:37,504 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:40,434 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:40,975 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:41,416 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:41,844 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:42,385 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:42,504 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:45,434 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:45,975 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:46,416 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:46,845 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:47,386 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:47,505 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:50,434 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:50,976 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:51,417 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:51,845 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:52,387 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:52,505 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:55,435 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:55,977 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:56,417 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:56,846 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:11:57,388 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:11:57,506 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:00,436 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:00,977 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:01,418 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:01,847 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:02,389 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:12:02,389 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:12:02,506 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:02,700 INFO    StreamThr :3613517 [internal.py:wandb_internal():86] W&B internal server running at pid: 3613517, started at: 2025-04-09 16:12:02.700096
2025-04-09 16:12:02,701 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status
2025-04-09 16:12:02,703 INFO    WriterThread:3613517 [datastore.py:open_for_write():85] open: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/5/wandb/run-20250409_161202-t3ftzaur/run-t3ftzaur.wandb
2025-04-09 16:12:02,704 DEBUG   SenderThread:3613517 [sender.py:send():380] send: header
2025-04-09 16:12:02,706 DEBUG   SenderThread:3613517 [sender.py:send():380] send: run
2025-04-09 16:12:02,853 ERROR   SenderThread:3613517 [internal_api.py:execute():369] 403 response executing GraphQL.
2025-04-09 16:12:02,853 ERROR   SenderThread:3613517 [internal_api.py:execute():370] {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-04-09 16:12:02,853 ERROR   SenderThread:3613517 [sender.py:send_run():977] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 975, in send_run
    self._init_run(run, config_value_dict)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2007, in upsert_run
    response = self.gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 964, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-04-09 16:12:03,142 INFO    StreamThr :3613517 [internal.py:wandb_internal():86] W&B internal server running at pid: 3613517, started at: 2025-04-09 16:12:03.141872
2025-04-09 16:12:03,142 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status
2025-04-09 16:12:03,147 INFO    WriterThread:3613517 [datastore.py:open_for_write():85] open: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/6/wandb/run-20250409_161203-tmxuq5k9/run-tmxuq5k9.wandb
2025-04-09 16:12:03,149 DEBUG   SenderThread:3613517 [sender.py:send():380] send: header
2025-04-09 16:12:03,150 DEBUG   SenderThread:3613517 [sender.py:send():380] send: run
2025-04-09 16:12:03,275 ERROR   SenderThread:3613517 [internal_api.py:execute():369] 403 response executing GraphQL.
2025-04-09 16:12:03,275 ERROR   SenderThread:3613517 [internal_api.py:execute():370] {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-04-09 16:12:03,276 ERROR   SenderThread:3613517 [sender.py:send_run():977] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 975, in send_run
    self._init_run(run, config_value_dict)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2007, in upsert_run
    response = self.gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 964, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-04-09 16:12:03,584 INFO    StreamThr :3613517 [internal.py:wandb_internal():86] W&B internal server running at pid: 3613517, started at: 2025-04-09 16:12:03.583819
2025-04-09 16:12:03,584 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status
2025-04-09 16:12:03,587 INFO    WriterThread:3613517 [datastore.py:open_for_write():85] open: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/7/wandb/run-20250409_161203-v2qxwk9z/run-v2qxwk9z.wandb
2025-04-09 16:12:03,588 DEBUG   SenderThread:3613517 [sender.py:send():380] send: header
2025-04-09 16:12:03,590 DEBUG   SenderThread:3613517 [sender.py:send():380] send: run
2025-04-09 16:12:03,780 ERROR   SenderThread:3613517 [internal_api.py:execute():369] 403 response executing GraphQL.
2025-04-09 16:12:03,780 ERROR   SenderThread:3613517 [internal_api.py:execute():370] {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-04-09 16:12:03,780 ERROR   SenderThread:3613517 [sender.py:send_run():977] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 975, in send_run
    self._init_run(run, config_value_dict)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2007, in upsert_run
    response = self.gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 964, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-04-09 16:12:05,436 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:05,978 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:06,419 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:06,847 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:07,507 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:07,854 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:08,277 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:08,591 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:08,781 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:10,437 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:10,979 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:11,419 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:11,848 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:12,507 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:12,855 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:13,277 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:13,592 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:13,782 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:15,438 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:15,979 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:16,420 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:16,848 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:17,508 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:17,855 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:18,278 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:18,593 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:18,782 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:20,438 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:20,980 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:21,421 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:21,849 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:22,509 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:22,856 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:23,279 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:23,594 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:23,783 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:25,439 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:25,981 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:26,421 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:26,850 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:27,510 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:27,857 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:28,280 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:28,595 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:28,784 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:30,439 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:30,981 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:31,422 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:31,850 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:32,511 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:32,857 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:33,280 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:33,596 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:33,784 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:35,440 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:35,982 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:36,423 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:36,851 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:37,511 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:37,858 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:38,281 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:38,597 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:38,785 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:40,440 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:40,983 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:41,423 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:41,851 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:42,512 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:42,858 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:43,282 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:43,597 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:43,785 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:45,441 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:45,983 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:46,424 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:46,852 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:47,513 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:47,859 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:48,282 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:48,598 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:48,786 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:50,441 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:50,984 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:51,425 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:51,853 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:52,514 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:52,860 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:53,283 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:53,599 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:53,786 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:55,442 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:55,984 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:56,426 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:56,853 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:57,515 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:57,861 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:58,284 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:12:58,600 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:12:58,787 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:00,442 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:00,985 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:01,426 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:01,854 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:02,515 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:02,861 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:03,284 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:03,601 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:03,788 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:05,443 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:05,985 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:06,427 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:06,854 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:07,516 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:07,861 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:08,285 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:08,602 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:08,788 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:10,444 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:10,986 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:11,427 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:11,855 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:12,516 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:12,862 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:13,285 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:13,603 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:13,789 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:15,444 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:15,986 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:16,428 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:16,855 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:17,517 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:17,862 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:18,286 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:18,604 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:18,790 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:20,445 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:20,987 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:21,429 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:21,856 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:22,517 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:22,863 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:23,286 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:23,605 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:23,790 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:25,446 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:25,988 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:26,430 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:26,857 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:27,518 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:27,863 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:28,287 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:28,606 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:28,791 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:30,446 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:30,988 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:31,430 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:31,858 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:32,519 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:32,864 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:33,288 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:33,607 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:13:33,607 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:13:33,792 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:33,919 INFO    StreamThr :3613517 [internal.py:wandb_internal():86] W&B internal server running at pid: 3613517, started at: 2025-04-09 16:13:33.919006
2025-04-09 16:13:33,920 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status
2025-04-09 16:13:33,922 INFO    WriterThread:3613517 [datastore.py:open_for_write():85] open: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/8/wandb/run-20250409_161333-4ztapm1m/run-4ztapm1m.wandb
2025-04-09 16:13:33,923 DEBUG   SenderThread:3613517 [sender.py:send():380] send: header
2025-04-09 16:13:33,925 DEBUG   SenderThread:3613517 [sender.py:send():380] send: run
2025-04-09 16:13:34,059 ERROR   SenderThread:3613517 [internal_api.py:execute():369] 403 response executing GraphQL.
2025-04-09 16:13:34,060 ERROR   SenderThread:3613517 [internal_api.py:execute():370] {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-04-09 16:13:34,060 ERROR   SenderThread:3613517 [sender.py:send_run():977] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 975, in send_run
    self._init_run(run, config_value_dict)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2007, in upsert_run
    response = self.gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 964, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-04-09 16:13:35,447 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:35,989 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:36,431 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:36,859 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:37,519 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:37,865 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:38,288 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:38,793 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:38,926 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:39,061 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:40,447 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:40,990 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:41,431 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:41,859 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:42,520 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:42,865 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:43,289 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:43,793 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:43,927 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:44,062 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:45,448 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:45,990 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:46,432 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:46,860 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:47,521 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:47,866 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:48,290 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:48,794 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:48,928 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:49,063 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:50,448 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:50,991 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:51,433 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:51,861 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:52,521 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:52,866 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:53,290 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:53,795 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:53,929 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:54,063 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:55,449 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:55,992 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:56,433 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:56,861 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:57,522 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:57,867 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:58,291 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:58,795 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:13:58,930 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:13:59,064 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:00,449 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:00,993 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:01,434 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:01,862 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:02,522 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:02,867 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:03,292 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:03,796 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:03,931 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:04,065 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:05,450 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:05,993 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:06,434 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:06,863 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:07,523 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:07,868 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:08,292 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:08,797 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:08,931 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:09,065 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:10,450 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:10,994 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:11,435 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:11,863 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:12,524 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:12,868 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:13,293 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:13,798 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:13,932 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:14,066 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:15,451 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:15,994 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:16,436 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:16,864 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:17,525 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:17,869 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:18,294 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:18,798 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:18,933 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:19,066 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:20,451 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:20,995 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:21,436 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:21,865 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:22,525 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:22,869 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:23,294 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:23,799 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:23,934 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:24,067 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:25,452 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:25,996 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:26,437 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:26,865 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:27,526 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:27,870 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:28,295 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:28,799 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:28,935 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:29,068 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:30,453 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:30,996 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:31,437 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:31,865 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:32,526 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:32,870 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:33,295 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:33,800 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:33,936 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:34,068 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:35,453 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:35,997 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:36,438 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:36,866 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:37,527 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:37,871 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:38,296 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:38,801 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:38,937 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:39,069 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:40,454 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:40,998 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:41,439 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:41,867 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:42,528 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:42,871 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:43,297 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:43,801 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:43,938 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:44,069 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:45,454 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:45,998 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:46,439 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:46,868 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:47,528 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:47,872 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:48,297 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:48,802 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:48,939 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:49,070 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:50,454 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:50,999 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:51,440 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:51,868 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:52,529 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:52,873 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:53,298 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:53,802 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:53,939 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:54,070 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:55,455 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:55,999 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:56,440 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:56,869 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:57,530 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:57,873 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:58,299 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:58,803 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:14:58,940 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:14:59,071 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:00,455 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:01,000 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:01,441 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:01,869 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:02,530 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:02,874 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:03,299 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:03,803 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:03,941 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:15:03,941 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:15:04,072 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:04,391 INFO    StreamThr :3613517 [internal.py:wandb_internal():86] W&B internal server running at pid: 3613517, started at: 2025-04-09 16:15:04.391541
2025-04-09 16:15:04,393 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status
2025-04-09 16:15:04,396 INFO    WriterThread:3613517 [datastore.py:open_for_write():85] open: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/9/wandb/run-20250409_161504-dewqkgi7/run-dewqkgi7.wandb
2025-04-09 16:15:04,397 DEBUG   SenderThread:3613517 [sender.py:send():380] send: header
2025-04-09 16:15:04,400 DEBUG   SenderThread:3613517 [sender.py:send():380] send: run
2025-04-09 16:15:04,563 ERROR   SenderThread:3613517 [internal_api.py:execute():369] 403 response executing GraphQL.
2025-04-09 16:15:04,563 ERROR   SenderThread:3613517 [internal_api.py:execute():370] {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-04-09 16:15:04,563 ERROR   SenderThread:3613517 [sender.py:send_run():977] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 975, in send_run
    self._init_run(run, config_value_dict)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2007, in upsert_run
    response = self.gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 964, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-04-09 16:15:05,456 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:06,000 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:06,441 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:06,870 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:07,531 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:07,874 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:08,300 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:08,804 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:09,072 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:09,400 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:09,564 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:10,456 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:11,001 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:11,442 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:11,870 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:12,531 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:12,875 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:13,300 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:13,805 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:14,073 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:14,401 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:14,565 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:15,457 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:16,001 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:16,442 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:16,871 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:17,532 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:17,875 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:18,301 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:18,805 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:19,073 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:19,402 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:19,565 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:20,457 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:21,002 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:21,443 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:21,871 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:22,532 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:22,876 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:23,301 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:23,806 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:24,074 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:24,403 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:24,566 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:25,458 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:26,002 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:26,443 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:26,872 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:27,533 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:27,876 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:28,302 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:28,806 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:29,075 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:29,404 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:29,566 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:30,458 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:31,003 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:31,444 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:31,872 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:32,534 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:32,877 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:33,302 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:33,807 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:34,075 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:34,404 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:34,567 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:35,459 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:36,004 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:36,444 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:36,873 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:37,534 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:37,877 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:38,303 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:38,807 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:39,076 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:39,405 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:39,568 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:40,459 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:41,004 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:41,445 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:41,874 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:42,535 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:42,878 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:43,303 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:43,808 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:44,076 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:44,406 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:44,568 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:45,460 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:46,005 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:46,446 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:46,874 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:47,535 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:47,878 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:48,304 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:48,808 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:49,077 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:49,407 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:49,569 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:50,461 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:51,006 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:51,446 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:51,875 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:52,536 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:52,879 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:53,305 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:53,809 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:54,078 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:54,408 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:54,570 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:55,461 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:56,006 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:56,447 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:56,876 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:57,537 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:57,879 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:58,305 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:58,810 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:59,078 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:15:59,409 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:15:59,570 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:00,462 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:01,007 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:01,448 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:01,877 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:02,537 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:02,880 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:03,306 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:03,811 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:04,079 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:04,410 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:04,571 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:05,462 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:06,007 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:06,448 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:06,877 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:07,538 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:07,880 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:08,307 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:08,811 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:09,080 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:09,411 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:09,572 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:10,463 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:11,008 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:11,449 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:11,878 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:12,538 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:12,881 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:13,307 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:13,812 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:14,080 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:14,412 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:14,572 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:15,463 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:16,009 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:16,450 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:16,879 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:17,539 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:17,881 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:18,308 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:18,813 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:19,081 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:19,413 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:19,573 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:20,464 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:21,009 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:21,450 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:21,880 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:22,540 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:22,882 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:23,309 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:23,813 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:24,081 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:24,414 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:24,574 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:25,464 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:26,010 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:26,451 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:26,880 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:27,540 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:27,882 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:28,309 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:28,814 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:29,082 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:29,415 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:29,574 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:30,465 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:31,010 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:31,451 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:31,881 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:32,541 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:32,883 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:33,310 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:33,814 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:34,083 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:34,415 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:16:34,416 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: cancel
2025-04-09 16:16:34,575 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:35,144 INFO    StreamThr :3613517 [internal.py:wandb_internal():86] W&B internal server running at pid: 3613517, started at: 2025-04-09 16:16:35.144071
2025-04-09 16:16:35,145 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status
2025-04-09 16:16:35,148 INFO    WriterThread:3613517 [datastore.py:open_for_write():85] open: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/10/wandb/run-20250409_161635-gdgzah9l/run-gdgzah9l.wandb
2025-04-09 16:16:35,148 DEBUG   SenderThread:3613517 [sender.py:send():380] send: header
2025-04-09 16:16:35,151 DEBUG   SenderThread:3613517 [sender.py:send():380] send: run
2025-04-09 16:16:35,465 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:35,569 ERROR   SenderThread:3613517 [internal_api.py:execute():369] 403 response executing GraphQL.
2025-04-09 16:16:35,569 ERROR   SenderThread:3613517 [internal_api.py:execute():370] {"errors":[{"message":"permission denied","path":["upsertBucket"],"extensions":{"code":"PERMISSION_ERROR"}}],"data":{"upsertBucket":null}}
2025-04-09 16:16:35,569 ERROR   SenderThread:3613517 [sender.py:send_run():977] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 975, in send_run
    self._init_run(run, config_value_dict)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 1013, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2007, in upsert_run
    response = self.gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 964, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/ghazal/anaconda3/lib/python3.10/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2025-04-09 16:16:36,011 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:36,452 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:36,881 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:37,541 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:37,886 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:38,310 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:38,816 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:39,083 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:39,576 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:40,152 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:40,466 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:40,570 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:41,011 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:41,453 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:41,882 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:42,542 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:42,887 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:43,311 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:43,816 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:44,084 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:44,576 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:45,153 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:45,466 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:45,571 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:46,012 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:46,453 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:46,882 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:47,542 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:47,887 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:48,312 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:48,817 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:49,084 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:49,577 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:50,154 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:50,467 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:50,571 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:51,013 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:51,454 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:51,883 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:52,543 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:52,888 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:53,312 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:53,817 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:54,085 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:54,577 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:55,154 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:16:55,467 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:55,572 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:56,013 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:56,454 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:56,883 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:57,543 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:57,888 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:58,313 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:58,818 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:59,085 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:16:59,578 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:17:00,155 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: keepalive
2025-04-09 16:17:00,468 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:17:00,513 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: shutdown
2025-04-09 16:17:00,513 INFO    HandlerThread:3613517 [handler.py:finish():866] shutting down handler
2025-04-09 16:17:00,572 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:17:01,014 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:17:01,455 DEBUG   HandlerThread:3613517 [handler.py:handle_request():146] handle_request: status_report
2025-04-09 16:17:01,468 INFO    SenderThread:3613517 [sender.py:finish():1534] shutting down sender
2025-04-09 16:17:01,469 INFO    WriterThread:3613517 [datastore.py:close():294] close: /home/ghazal/Documents/NS_Projects/NS_Revised_P1/neural_decoding/outputs/logs/multirun/2025-04-09_16-10-27/0/wandb/run-20250409_161030-tj9k6r3d/run-tj9k6r3d.wandb
